astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -1883,40 +1822,50 @@ Each possible value <InlineField>error</InlineField> and <InlineField>error_reas
     * `change_password_validation`
     * `client_authentication_missing`
     * `client_id_mismatch`
+    * `consent_canceled`
     * `grant_type_disabled`
     * `invalid_additional_client_id`
-    * `invalid_client_authentication_scheme`
     * `invalid_client_authentication`
+    * `invalid_client_authentication_scheme`
     * `invalid_client_id`
     * `invalid_device_code`
+    * `invalid_entity_permission_scope`
     * `invalid_grant_type`
     * `invalid_id_token_hint`
     * `invalid_origin`
-    * `invalid_pkce_code_challenge_method`
```

Comment:
Is that a breaking change?

Reply:
No. I just moved it down two rows to be in alphabetical order.